azure-pipelines/templates/steps/nightly-snapshot-releases/common.yml

```diff
@@ -0,0 +1,71 @@
+# File: azure-pipelines\templates\steps\nightly-snapshot-releases\common.yml
+# Description: Template to assemble and publish different versions of common
+# Variable: 'ENV_VSTS_MVN_ANDROIDCOMMON_USERNAME' was defined in the Variables tab
+# Variable: 'mvnAccessToken' was defined in the Variables tab
+# https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate
+# Variable: 'versionType' type of version (Latest, Snapshot)
+
+parameters:
+- name: versionType
+  default: Latest
+  values:
+  - Snapshot
+  - Latest
+
+jobs:
+- job: Phase_1
+  displayName: Phase 1
+  timeoutInMinutes: 90
+  cancelTimeoutInMinutes: 1
+  pool:
+    name: Hosted VS2017
```

Comment:
>name: Hosted VS2017 [](http://example.com/codeflow?start=4&length=19)

I saw in some other pipeliens we are using "Hosted Windows 2019 with VS2019" ... Is it because we dont have enough hosts in each pool? or those require Server2019?

Reply:
not sure.. good catch, my guess is using VS2017 because it is an older pipeline, will update and test